js/index.js

```diff
@@ -16,3 +16,181 @@ menuBtn.addEventListener('click', () => {
+  liveLink,
+  sourceCode,
+) {
+  return `
+  <section class="modal-mobile" id="popup-project-${id}">
+        <div class="modal-head">
+          <h2 class="title">${title}</h2>
+          <img
+            src="assets/images/close-window.svg"
+            alt="close-window"
+            class="close-window"
+            id="close-${id}"
+          />
+        </div>
+        <ul class="categories">
+          <li class="item1">${company}</li>
+          <li class="counter">
+            <img
+              class="img-counter"
+              src="assets/images/Counter.svg"
+              alt="Counter"
+            />${role}
+          </li>
+          <li class="counter">
+            <img
+              class="img-counter"
+              src="assets/images/Counter.svg"
+              alt="Counter"
+            />${year}
+          </li>
+        </ul>
+        <img
+          src="${img1}"
+          class="project-1 project"
+          alt="project-one"
+        />
+
+        <img
+          src="${img}"
+          class="project-1 project-1-desktop"
+          id="project1-desktop"
+          alt="project-one"
+        />
+        <div class="project-wrapper">
+          <article class="intro-works">
+           ${description}
+          </article>
+          <div class="lang-buttons">
+            <ul class="languages">
+            ${languages.map((item) => `<li>${item}</li>`).join('')}
+            </ul>
+            <img src="assets/images/vector 4.svg" alt="line" class="line" />
+            <div class="window-buttons">
+              <button class="window-btn">
+              <a href='${liveLink}' class="PopUp-link">
+               See live
+                <span><img src="assets/images/live.svg" alt="live" /></span>
+                
+              </button>
+              <button class="window-btn">
+              <a href='${sourceCode}' class="PopUp-link">
+                See source
+                <span
+                  ><img src="assets/images/git-modal.svg" alt="source code"
+                /></span>
+              </a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </section>
+  `;
+}
+
+let final = '';
+data.forEach((item) => {
+  const html = createPopUp(
+    item.id,
+    item.title,
+    item.company,
+    item.description,
+    item.role,
+    item.year,
+    item.img1,
+    item.img,
+    item.languages,
+    item.liveLink,
+    item.sourceCode,
+  );
+  final += html;
+});
+
+document.getElementById('popup-container').innerHTML = final;
+
+// event listeners
+data.forEach((item) => {
+  // open event
+  const { id } = item;
+  document.getElementById(`openpop-${id}`).addEventListener('click', () => {
+    document.getElementById(`popup-project-${id}`).classList.add('visible');
+  });
+  // close event
+  document.getElementById(`close-${id}`).addEventListener('click', () => {
+    document.getElementById(`popup-project-${id}`).classList.remove('visible');
+  });
+});
```

Comment:
- [ ] When the application is started, the popup window is loaded 😞 . This does not match the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#project-requirements). The project requirements suggest that when the user clicks (or taps) the button to check project details, the popup with details about the project appears. When the user clicks (or taps) the close (X) button, the popup disappears. Kindly ensure to apply this, so that the project performs as intended ☺️ .